GUI/Types/Renderer/SelectedNodeRenderer.cs

```diff
@@ -63,6 +66,7 @@ public void SelectNode(SceneNode? node, bool forceDisableDepth = false)
             }
 
             selectedNodes.Add(node);
+            node.IsSelected = true;
```

Comment:
This class could fill the renderlists directly? It would need to check if the node passed frustum cull.

Reply:
I mean we already fill `selectedNodes`, could use that?